Makefile

```diff
@@ -3,21 +3,56 @@ SHELL := /bin/bash # Use bash syntax
 # Set up variables
 GO111MODULE=on
 
-# Build ldflags
-VERSION ?= "v0.0.0"
-GITCOMMIT=$(shell git rev-parse HEAD)
-BUILDDATE=$(shell date -u +'%Y-%m-%dT%H:%M:%SZ')
-GO_LDFLAGS=-ldflags "-X main.version=$(VERSION) \
-			-X main.buildHash=$(GITCOMMIT) \
-			-X main.buildDate=$(BUILDDATE)"
```

Comment:
I think we will still want to keep these variables, in case we want to version the controller bootstrap CLI in the future

Reply:
Thank you for pointing this out, the variables have been added to the file.